src/_msc/print-shapes.py

```diff
@@ -0,0 +1,283 @@
+import argparse
```

Comment:
Hi @thangk 
Not sure this long lines of code is necessary for a simple comparison check of two arrays row by row ...

Reply:
yes, I meant the entire file. pls merge it into the main file for less than 10-20 lines in total